src/loan-returned/handler.js

```diff
@@ -0,0 +1,21 @@
+const extractMessageData = require('../extract-message-data')
+const Cache = require('../cache')
+
+module.exports.handle = (event, context, callback) => {
+  let loanData
+
+  let almaCache = new Cache({
```

Comment:
Can this be extracted out external to this that when required returns an instance of the almaCache from the environment? Would help reduce duplication